frontend/src/main/scala/bloop/cli/CliOptions.scala

```diff
@@ -14,6 +14,9 @@ case class CliOptions(
     version: Boolean = false,
     @HelpMessage("If set, print out debugging information to stderr.")
     verbose: Boolean = false,
+    @ExtraName("f")
+    @HelpMessage("If set, print debug messages that contain given string.")
+    filter: String = "",
```

Comment:
Would it be possible to use `bloop compile foo | grep filter` instead?

Reply:
Good question, it got me thinking.
  
@Duhemm made the point that this is less than ideal when the log messages use more than one line (which not only happens for compiler messages but for other bloop logs). I think this is a good argument to merit a filtering option on its own.